web/admin/application/languages/es_ES/es_ES.po

```diff
@@ -1085,6 +1086,9 @@ msgstr "Prioridad"
 msgid "Matching type"
 msgstr "Tipo coincidencia"
 
+msgid "Max call"
+msgstr "Límite llamadas"
+
 msgid "Max calls"
 msgstr "Límite llamadas"
 
```

Comment:
I would suggest using the already existing text _Max calls_ instead of creating a new one.